src/masonite/routes/Router.py

```diff
@@ -49,6 +57,24 @@ def preflight_response(app):
         else:
             raise MethodNotAllowedException(matched_methods, request_method)
 
+    def get_fallback_route(self):
+        """
+        This method is used to get the fallback route from the list of routes.
+
+        The fallback route is the one where the `is_fallback` attribute is True.
+        If there are multiple fallback routes, it returns the first one.
+        If there are no fallback routes, it returns an empty list.
+
+        Returns:
+            fallback_route (Route): The fallback route if it exists, otherwise an empty list.
+        """
+        fallback_route = list(filter(lambda r: r.is_fallback, self.routes))
```

Comment:
i would imagine this would be a pretty slow way to get a single route. filtering a list just to find the fallback route ..

can we just set this on the router class and just do like 

```python
fallback_route = self.fallback_route
```

Reply:
actually nvm its only if theres no route found so its probable fine